config/airflow.cfg

```diff
@@ -153,6 +150,9 @@ dag_run_conf_overrides_params = False
 # Worker initialisation check to validate Metadata Database connection
 worker_precheck = False
 
+# When discovering DAGs, ignore any files that don't contain the strings `DAG` and `airflow`.
+dag_discovery_safe_mode = False
```

Comment:
The default for this used to be True btw.

Reply:
Just pushed change to fix that.